restcomm/restcomm.docs/sources-asciidoc/src/main/asciidoc/release-notes.adoc

```diff
@@ -0,0 +1,47 @@
+= Restcomm Release Notes
+
+These notes group both Community and Telscale changes in a single document.
```

Comment:
"telscale" => "Commercial" ?

Reply:
Changed that to `RestcommOne` and also added a new section to explain the tags used in the notes (see below)